src/views/HomeView/HomeView.js

```diff
@@ -13,7 +13,7 @@ import classes from './HomeView.scss'
 const mapStateToProps = (state) => ({
   counter: state.counter
 })
-export class HomeView extends React.Component {
+class HomeView extends React.Component {
```

Comment:
tracking thejameskyle/babel-plugin-react-pure-components#5

export syntax (on line 54 below) is slightly modified awaiting hotfix above—will be changed back to a named es module export (`export class HomeView extends React.Component {}`) as soon as pr above is merged